sdk/trace/src/test/scala/org/typelevel/otel4s/sdk/trace/SdkSpanBackendSuite.scala

```diff
@@ -218,15 +218,23 @@ class SdkSpanBackendSuite extends CatsEffectSuite with ScalaCheckEffectSuite {
         )
 
         _ <- assertIO(
-          extra.toList.traverse(a => span.getAttribute(a.key)),
+          extra.view
+            .map(_.key)
+            .filterNot(init.contains)
+            .toList
+            .traverse(span.getAttribute(_)),
```

Comment:
I do not know why this needs `(_)` on `span.getAttribute`, but intellij insists it does